anaconda_example/README.md

```diff
@@ -0,0 +1,22 @@
+# Introduction
+This barebones example shows how to run TensorFlow GPU jobs on condor without using a Docker container. It uses Anaconda to install the necessary cuda libraries. Simply upload to a submit node and call `condor_submit gpu_job.sub` to run the example.
+
+# Supported Servers
+This example uses a whitelist and supports different environment setups depending on the specific server it is running on. Currently, the following servers are supported:
```

Comment:
Any way to identify a configuration (machine attributes in HTCondor like CUDA version) that would be more generalizable than using specific server names? At some point, any of the below servers might disappear, or their name or configuration may change, or we will have added new machines that would work but are not in the whitelist. If you don't think it would be possible to replace the whitelist with generalizable requirements for machine attributes, we'd like to understand why and 'fix' that.